README.md

```diff
@@ -17,6 +17,8 @@ _Agency Website for TECVITY_
   - [Email Configuration](#email-configuration)
   - [Database](#database)
   - [Disabled Features](#disabled-features)
+  - [Blog Post Contriution Guide](#blog-post-contribution-guide)
+  - [Service Addition Guide](#service-addition-guide)
```

Comment:
why is this added? only add when the section we're mentioning is also added

Reply:
added by mistake, removing in the next commit.